webui/src/js/windowStateUtils.js

```diff
@@ -234,6 +267,19 @@ function(wktProject, wktConsole, wdtDiscoverer, dialogHelper, projectIO,
     });
   }
 
+  /**
+   * Remove keyboard focus from the active DOM element.
+   * This causes Ace editor and Jet controls to persist their current values.
+   * This workaround is required because the menu-will-show event on the application menu
+   * does not fire correctly on MacOS, so the blur-focused-item IPC is not sent to the renderer.
+   * See GitHub electron issue 31915.
+   */
+  function blurSelection() {
+    if(document.activeElement) {
```

Comment:
I would like to wrap this in an if statement so that it only executes on MacOS.

```javascript
if (window.api.process.isMac()) {
    ...
}
```

Reply:
Done